app/pages/sections/supplier/select/contextCreator.js

```diff
@@ -0,0 +1,29 @@
+import manifest from './manifest.json';
+import { baseUrl } from '../../../../config';
+
+const generateSupplierOptions = ({ suppliers }) => {
+  const supplierOptions = suppliers.map(supplier => ({
```

Comment:
can just return from this line:
const generateSupplierOptions = ({ suppliers }) => suppliers.map(supplier => ({
    value: supplier.supplierId,
    text: supplier.name,
  }));